.env.example

```
@@ -3,13 +3,23 @@
 # ===========================================
 # Copy this file to .env and fill in your actual values
 
+ENVIRONMENT=development
+DEBUG_MODE=true
+LOG_LEVEL=INFO
+LOG_FORMAT=json
+DATA_PATH=../data
+
 # Database Configuration
 DB_CONNECTION_STRING=Server=host.docker.internal,1433;Database=ProjectVG;User Id=sa;Password=YOUR_DB_PASSWORD;TrustServerCertificate=true;MultipleActiveResultSets=true
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)